localstack/services/apigateway/helpers.py

```diff
@@ -673,7 +673,14 @@ def add_path_methods(rel_path: str, parts: List[str], parent_id=""):
         paths_dict = resolved_schema["paths"]
         method_paths = paths_dict.get(rel_path, {})
         for field, field_schema in method_paths.items():
-            if field in ["parameters", "servers", "description", "summary", "$ref"]:
+            if field in [
+                "parameters",
+                "servers",
+                "description",
+                "summary",
+                "$ref",
+            ] and not isinstance(field_schema, dict):
```

Comment:
Should this be an `or` instead of `and` here? Sorry, didn't make myself clear in the previous comment. I think we want to `continue` here if either the `field` is in the list of field names, or if `field_schema` is not a `dict` (to avoid errors when accessing `field_schema.get(..)` in line 688 below. Thoughts @calvernaz ?

Reply:
Makes sense, pushing the change